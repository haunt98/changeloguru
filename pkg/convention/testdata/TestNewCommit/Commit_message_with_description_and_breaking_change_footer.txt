feat: allow provided config object to extend other configs

BREAKING CHANGE: `extends` key in config file is now used for extending other config files
