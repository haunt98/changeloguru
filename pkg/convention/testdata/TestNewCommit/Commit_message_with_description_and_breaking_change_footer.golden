{
  "Type": "feat",
  "Scope": "",
  "Description": "allow provided config object to extend other configs",
  "BodyAndFooters": "BREAKING CHANGE: `extends` key in config file is now used for extending other config files"
}